bundle/bundle.go

```diff
@@ -8,23 +8,26 @@ import (
 	"os"
 	"strings"
 
+	"github.com/deislabs/cnab-go/bundle/definition"
 	"github.com/docker/go/canonical/json"
+	pkgErrors "github.com/pkg/errors"
 )
 
 // Bundle is a CNAB metadata document
 type Bundle struct {
-	SchemaVersion    string                `json:"schemaVersion" mapstructure:"schemaVersion"`
-	Name             string                `json:"name" mapstructure:"name"`
-	Version          string                `json:"version" mapstructure:"version"`
-	Description      string                `json:"description" mapstructure:"description"`
-	Keywords         []string              `json:"keywords,omitempty" mapstructure:"keywords"`
-	Maintainers      []Maintainer          `json:"maintainers,omitempty" mapstructure:"maintainers"`
-	InvocationImages []InvocationImage     `json:"invocationImages" mapstructure:"invocationImages"`
-	Images           map[string]Image      `json:"images" mapstructure:"images"`
-	Actions          map[string]Action     `json:"actions,omitempty" mapstructure:"actions"`
-	Parameters       ParametersDefinition  `json:"parameters" mapstructure:"parameters"`
-	Credentials      map[string]Credential `json:"credentials" mapstructure:"credentials"`
-	Outputs          OutputsDefinition     `json:"outputs" mapstructure:"outputs"`
+	SchemaVersion    string                 `json:"schemaVersion" mapstructure:"schemaVersion"`
+	Name             string                 `json:"name" mapstructure:"name"`
+	Version          string                 `json:"version" mapstructure:"version"`
+	Description      string                 `json:"description" mapstructure:"description"`
+	Keywords         []string               `json:"keywords,omitempty" mapstructure:"keywords"`
+	Maintainers      []Maintainer           `json:"maintainers,omitempty" mapstructure:"maintainers"`
+	InvocationImages []InvocationImage      `json:"invocationImages" mapstructure:"invocationImages"`
+	Images           map[string]Image       `json:"images" mapstructure:"images"`
+	Actions          map[string]Action      `json:"actions,omitempty" mapstructure:"actions"`
+	Parameters       ParametersDefinition   `json:"parameters" mapstructure:"parameters"`
+	Credentials      map[string]Credential  `json:"credentials" mapstructure:"credentials"`
+	Outputs          OutputsDefinition      `json:"outputs" mapstructure:"outputs"`
+	Definitions      definition.Definitions `json:"definitions" mapstructure:"definitions"`
```

Comment:
Is this section mandatory or optional? Should it be tagged as `omitempty` ?

Reply:
I'm not sure. Parameters and Outputs are also not included, but I think all three probably end up being non-required. I'll tag here and we can revisit the other two in a specific PR/issue combo?